From 3907ea187d001ab71b52ed658952411148a34015 Mon Sep 17 00:00:00 2001
From: D <stupid@unicode.com>
Date: Sun, 8 Dec 2019 08:05:11 +0000
Subject: [PATCH] fix unicode

---
 debian/copyright | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/copyright b/debian/copyright
index 6745f00..b5eb113 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -9511,7 +9511,7 @@ License: Ms-PL
  A "contributor" is any person that distributes its contribution under this
  license.
  .
- "Licensed patents" are a contributor’s patent claims that read directly on its
+ "Licensed patents" are a contributor's patent claims that read directly on its
  contribution.
  .
  .
@@ -9534,7 +9534,7 @@ License: Ms-PL
  3. Conditions and Limitations
  .
  (A) No Trademark License- This license does not grant you rights to use any
- contributors’ name, logo, or trademarks.
+ contributors' name, logo, or trademarks.
  .
  (B) If you bring a patent claim against any contributor over patents that you
  claim are infringed by the software, your patent license from such contributor
-- 
2.17.1

